src/main/java/org/jmxtrans/agent/ResultNameStrategyImpl.java

```diff
@@ -105,10 +105,12 @@
     public String getResultName(@Nonnull Query query, @Nonnull ObjectName objectName, @Nullable String key, @Nonnull String attribute) {
```

Comment:
Yes, you are right. That creates a backward compatibility issues. I will move this code to JConsoleNameStrategyImpl and remove any changes from ResultNameStrategyImpl